src/main/java/teammates/common/util/Const.java

```diff
@@ -275,6 +278,7 @@ private Const() {
         public static final String SESSION_STATS = URI_PREFIX + "/session/stats";
         public static final String SESSION_SUBMITTED_GIVER_SET = URI_PREFIX + "/session/submitted/giverset";
         public static final String SESSIONS = URI_PREFIX + "/sessions";
+        public static final String SESSION_ACCESS_LOGS = URI_PREFIX + "/sesion/accesslogs";
```

Comment:
I was thinking access can be either a visit or a submission. I thought that just calling it logs might be misleading since its only the logs that we are writing and not the request logs. I will change to logs as it makes more sense.